CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the "vscode-extension-for-zowe" extension will be documented in this file.
 
+## `1.10.0`
+
+- Added Base Profile support. [#1037](https://github.com/zowe/vscode-extension-for-zowe/issues/1037). Thanks @katelynienaber, @jellypuno, @JillieBeanSim, @lauren-li, @crawr, @phaumer
```

Comment:
You can now log in to the API ML via the CLI and use that token with a service profile in Zowe Explorer. 

Something like that to add?

Reply:
Is that the only feature that has been added in this release?